Source/ToolCore/JSBind/JSBHeaderVisitor.h

```diff
@@ -402,7 +403,14 @@ class JSBHeaderVisitor : public SymbolVisitor
             return true;
         }
 
-        module_->RegisterConstant(getNameString(decl->name()).CString());
+        if (type->isIntegerType())
+        {
+            module_->RegisterConstant(getNameString(decl->name()).CString(), JSBPrimitiveType::Int);
+        }
+        else
+        {
+            module_->RegisterConstant(getNameString(decl->name()).CString(), JSBPrimitiveType::Float);
```

Comment:
Does this need to check for float or double?  Could it possibly be a string?  Maybe it should ignore if not a float/double?